src/mpc-hc/MainFrm.cpp

```diff
@@ -966,9 +977,23 @@ int CMainFrame::OnCreate(LPCREATESTRUCT lpCreateStruct)
 
     UpdateSkypeHandler();
 
+    m_popupMenu.fulfillThemeReqs();
+    m_mainPopupMenu.fulfillThemeReqs();
     return 0;
 }
 
+void CMainFrame::OnMeasureItem(int nIDCtl, LPMEASUREITEMSTRUCT lpMeasureItemStruct) {
+    if (lpMeasureItemStruct->CtlType == ODT_MENU)  {
+        if (CMPCThemeMenu* cm = CMPCThemeMenu::getParentMenu(lpMeasureItemStruct->itemID)) {
+            cm->MeasureItem(lpMeasureItemStruct);
+            return;
+        }
+

```

Comment:
Useless extra newline. In a few other places in this file's diff as well.

Reply:
Cleaned up.  I removed some other triple newlines that were already there, as well.